vendor.conf

```diff
@@ -1,7 +1,7 @@
 # the following lines are in sorted order, FYI
 github.com/Azure/go-ansiterm d6e3b3328b783f23731bc4d058875b0371ff8109
 github.com/Microsoft/hcsshim ada9cb39f715fb568e1030e7613732bb4f1e4aeb
-github.com/Microsoft/go-winio 4de24ed3e8c509e6d1f609a8cb6b1c9fd9816e6d
+github.com/Microsoft/go-winio c599b533b43b1363d7d7c6cfda5ede70ed73ff13
```

Comment:
@jhowardmsft do you know if a new tag will be created anytime soon? (so that we can have a tagged version in the 19.03 branch)

Reply:
@thaJeztah We're talking about the possibility of a tagged release soon, but there's still a bunch of fixes we're working on for CRI/containerd, many of which also affect docker scenarios, hence our reticence until we're in a more stable position.

Reply:
Gotcha! I'll await progress there 🤗